modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,158 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="support-matrix-for-primary-secondary-networks_{context}"]
+= Support matrix for primary and secondary networks
+
+The `UserDefinedNetwork` and `NetworkAttachmentDefinition` custom resources (CRs) provide cluster administrators and users the ability to create customizable network configurations and define their own network topologies, ensure network isolation, manage IP addressing for workloads, and leverage advanced network features. A third CR, `ClusterUserDefinedNetwork`, is also available, which allows administrators the ability to create and define additional networks spanning multiple namespaces at the cluster level. Both user-defined networks and network attachment definitions can serve as both the primary and secondary network interface, and each support `layer2` and `layer3` topologies; a third network topology, Localnet, is also supported with network attachment definitions with secondary networks. 
+
+[NOTE]
+====
+As of {product-title} 4.18, the Localnet topology is unavailable for use with the `UserDefinedNetwork` leveraging primary networks and `ClusterUserDefinedNetwork` APIs. It is available for use with network attachment definitions that leverage secondary networks only.
+====
+
+The following section highlights the supported features of the `UserDefinedNetwork` and `NetworkAttachmentDefinition` CRs when they are used as either the primary or secondary network. A separate table for the `ClusterUserDefinedNetwork` CR is also included. 
+ 
+.Primary network support matrix
+[cols="1,1a,1a,1a, options="header"]
+|===
+| Network type | Network feature | Layer2 topology |Layer3 topology
+
+
+.10+| Primary network (UDN and NAD)
+
+| *east-west traffic*
+^| &#10003;
+^| &#10003;
+
+| *north-south traffic*
+^| &#10003;
+^| &#10003;
+
+| *Persistent IPs*
+^| &#10003;
+^| X
+
+| *Services*
+^| &#10003;
+^| &#10003;
+
+| *`EgressIP` resource*
+^| &#10003;
+^| &#10003;
+
+| *Multicast* ^[1]^
+^| &#10003;
+^| &#10003;
+
+| *`NetworkPolicy` resource* ^[2]^
+^| &#10003;
+^| &#10003;
+
+| *`MultinetworkPolicy` resource* 
+^| X
+^| X
+
+|===
+1. Multicast must be enabled in the namespace, and it is only available between OVN-Kubernetes network pods. For more information about multicast, see "About multicast".
+2. When creating a `UserDefinedNetwork` CR with a primary network type, network policies must be created _after_ the `UserDefinedNetwork` CR.
+
+
+.Secondary network support matrix
+[cols="1,1a,1a,1a,1a, options="header"]
+|===
+| Network type | Network feature | Layer2 topology |Layer3 topology |Localnet topology ^[1]^
```

Comment:
Localnet topology will be supported after 4.18 GA

Reply:
Removed for now.

Reply:
@weliang1 Even for secondary networks?